pkg/apiserver/apiserver.go

```diff
@@ -27,8 +27,11 @@ import (
 	"path"
 	rt "runtime"
 	"strconv"
+	"strings"
 	"time"
 
+	"k8s.io/client-go/pkg/util/sets"
+	"k8s.io/client-go/pkg/version"
```

Comment:
Yes. I think these kind of utility packages should be moved to their own repo, then there will be only one copy of them.